src/site/antora/modules/ROOT/pages/manual/appenders/database.adoc

```diff
@@ -0,0 +1,1505 @@
+This allows you to optimize the unit for logging purposes.
+The definition of the persistence unit should look like the example below:
+
+[source,xml]
+----
+include::example$manual/appenders/database/persistence.xml[lines=1;18..-1]
+----
+
+<1> Specify you JPA provider.
+<2> A non-JTA source should be used for performance.
+<3> If your log event entity extends `BasicLogEventEntity`, you need to declare these converters.
+<4> Declare your log event entity.
+<5> Cache sharing should be set to `NONE`.
+
+[#JpaAppender-configuration]
+=== Appender configuration
+
+The JPA appender supports these configuration options:
+
+[#JpaAppender-attributes]
+.JPA Appender configuration attributes
+[cols="1m,2,1,5"]
+|===
+| Attribute | Type | Default value | Description
+
+4+h| Required
+
+| [[JpaAppender-attr-name]]name
+| `String`
+|
+| The name of the Appender.
+
+| [[JpaAppender-attr-tableName]]tableName
+| `String`
+|
+| The name of the table to use.
+
+| [[JpaAppender-attr-persistenceUnit]]persistenceUnitName
+| `String`
+|
+| The name of the persistence unit to use.
+
+| [[JpaAppender-attr-entityClassName]]entityClassName
+| https://docs.oracle.com/javase/8/docs/api/java/lang/Class.html[`Class<?>`]
+|
+| The fully qualified name of the entity class to use.
+
+The type must extend
+link:../javadoc/log4j-jpa/org/apache/logging/log4j/core/appender/db/jpa/AbstractLogEventWrapperEntity.html[`AbstractLogEventWrapperEntity`].
+
+4+h| Optional
+
+| [[JpaAppender-attr-bufferSize]]bufferSize
+| `int`
+| `0`
+|
+The number of log messages to batch before writing.
+If `0`, batching is disabled.
+
+| [[JpaAppender-attr-ignoreExceptions]]ignoreExceptions
+| `boolean`
+| `true`
+| If `false`, logging exception will be forwarded to the caller of the logging statement.
+Otherwise, they will be ignored.
+
+|===
+
+[#JpaAppender-elements]
+.JPA Appender nested elements
+[cols="1m,1,4"]
+|===
+| Type | Multiplicity | Description
+
+| [[JpaAppender-element-Filter]]xref:manual/filters.adoc[`Filter`]
+| zero or one
+|
+Allows filtering log events just before they are formatted and sent.
+
+See also xref:manual/filters.adoc#appender-stage[appender filtering stage].
+
+|===
+
+Additional runtime dependencies are required for using the Cassandra Appender:
+
+[tabs]
+====
+Maven::
++
+[source,xml]
+----
+<!-- We assume you use `log4j-bom` for dependency management -->
+<dependency>
+  <groupId>org.apache.logging.log4j</groupId>
+  <artifactId>log4j-jpa</artifactId>
+  <scope>runtime</scope>
+</dependency>
+----
+
+Gradle::
++
+[source,groovy]
+----
+// We assume you use `log4j-bom` for dependency management
+runtimeOnly 'org.apache.logging.log4j:log4j-jpa'
+----
+====
+
+xref:plugin-reference.adoc#org-apache-logging-log4j_log4j-jpa_org-apache-logging-log4j-core-appender-db-jpa-JpaAppender[{open-book} Plugin reference for `JPA`]
+
+[#JpaAppender-examples]
+=== Configuration examples
+
+[source,xml]
+----
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="error">
```

Comment:
This needs to be ported to `examples` in all supported configuration formats.

Reply:
Fixed in https://github.com/apache/logging-log4j2/pull/2790/commits/fad8d7f8603086a8abaa44c0c9e2d1dd6d7524fe